### DISCUSSION TITLE
Possible to disable issue writing?

### DISCUSSION THREAD

**Discussion by gooseops** [semantic-release/semantic-release#3366] [Q&A] [answered]
Currently hitting the Github API rate limit when these run in quick succession:

`Request quota exhausted for request GET /search/issues`

Is it possible to disable this part of the `semantic-release/github` plugin? Or can I change the `success` behavior of the plugin to do something different?

**Comment by travi** [comment, marked as answer]
you can find the options for the github plugin here: https://github.com/semantic-release/github#options

**Reply by gooseops** [reply]
Yes, I found those, but it wasn't immediately clear what to do with the options.  I cannot see a way to disable issue comment/closure in the plugin options, but I'm also ignorant sometimes :) .  Does this mean it is not possible to disable the Github issue closure using the options?

**Reply by gooseops** [reply]
Jeez, nvm.  I'm just blind.  

```
successComment	
The comment to add to each issue and pull request resolved by the release. Set to false to disable commenting on issues and pull requests. See [successComment](https://github.com/semantic-release/github?tab=readme-ov-file#successcomment).	:tada: This issue has been resolved in version ${nextRelease.version} :tada:\n\nThe release is available on [GitHub release](<github_release_url>)
```

**Reply by gooseops** [reply]
@travi 
I added the `"successComment: false` option to my `package.json`.  I am still receiving the quota limits errors

here is my `package.json`:
```
{
    "name": "foobar",
    "version": "0.0.0",
    "private": true,
    "release": {
        "branches": [
            "+([0-9])?(.{+([0-9]),x}).x",
            "master",
            "main",
            "next-major",
            {
                "name": "foobar",
                "prerelease": true
            },
            {
                "name": "main",
                "prerelease": true
            }
        ],
        "tagFormat": "v${version}"
    },
    "plugins": [
        "@semantic-release/commit-analyzer",
        "@semantic-release/release-notes-generator",
        "@semantic-release/npm",
        [
            "@semantic-release/github",
            {
                "options": {
                    "successComment": false
                }
            }
        ]
    ]
}
```

Please let me know what is incorrect or if I need to open an issue.

**Reply by pierredagorn** [reply]
I would love to disable the GET /search/issue to avoid the quota limit as well, too bad this option to false doesn't do the trick, I will follow this thread for sure!

**Reply by pierredagorn** [reply]
looks like it is solved by this:
https://github.com/semantic-release/semantic-release/issues/2204#issuecomment-2154938064
